tests/Unit/Evolution/DiscontinuousGalerkin/Initialization/Test_Mortars.cpp

```diff
@@ -142,6 +148,10 @@ void test_impl(const std::vector<std::array<size_t, Dim>>& initial_extents,
       CHECK(mortar_next_temporal_ids.at(mortar_id) == time_step_id);
     }
   }
+
+  CHECK(static_cast<bool>(
+      get_tag(evolution::dg::Tags::InternalFace::NormalCovectorAndMagnitude<
+              Dim>{}) == expected_normal_covector_quantities));
```

Comment:
Umm I think you misread the code. I cast the result of `operator==` to a bool because Catch complains about trying to stream some STL containers.

Reply:
I've added a code comment explaining what is happening and why

Reply:
whoops, I misread this a lot. Thanks for clarifying.

Nice trick, too... I think this actually might've come in useful in my PR to remove boost::optional, because I might've been able to avoid adding the std::optional stream operator by using this. But I figure it'll be nice to have anyway.

Reply:
Yea, but streaming STL containers is a nightmare because you aren't allowed to add the operators into the std namespace (reasonable), but ADL fails because the stream operators aren't in the std namespace. E.g. we can't stream a std container that holds a std container...